scss/_patterns_navigation.scss

```diff
@@ -190,6 +206,16 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
     justify-content: space-between;
     padding-right: 0;
 
+    .p-navigation__link {
+      // reser padding for navigation links in the navigation banner
+      @extend %navigation-link-responsive-padding-horizontal;
+      @extend %navigation-link-responsive-padding-vertical;
+
+      &::before {
```

Comment:
Could you add a line with a comment what is it resetting. It's a bit confusing when out of context of where the ::before comes from.